index.js

```diff
@@ -0,0 +1,87 @@
+import hide from './modules/hideHtml.js';
+import horaCurrente from './modules/calendar.js';
+
+class Book {
+  constructor() {
+    this.bookStore = JSON.parse(localStorage.getItem('singleBook')) || [];
+  }
+
+  getBookList() {
+    const bookList = document.getElementById('book-list');
+    bookList.innerHTML = '';
+    bookList.innerHTML += this.bookStore.map(
+      (el, index) => ` <div class="${index % 2 === 0 ? 'dark' : 'mediumdark'}">
+        <h4>${el.title}</h4>
+        <p>${el.author}</p>
+        <button type="button" id=${el.id} class='remove-button'>Remove</button>
+        </div>`,
+    );
+    const Allbtn = document.querySelectorAll('.remove-button');
+    Allbtn.forEach((btn) => {
+      btn.addEventListener('click', () => {
+        this.deleteBook(btn.id);
+      });
+    });
+  }
+
+  addNewBook(title, author) {
+    const id = this.bookStore.length + 1;
+    const addbook = {
+      title,
+      author,
+      id,
+    };
+    this.bookStore.push(addbook); localStorage.setItem('singleBook', JSON.stringify(this.bookStore));
+    this.getBookList();
+  }
+
+  deleteBook(id) {
+    this.bookStore = this.bookStore.filter((item) => {
+      if (item.id === Number(id)) {
+        return false;
+      }
+      return true;
+    });
+    localStorage.setItem('singleBook', JSON.stringify(this.bookStore));
+    this.getBookList();
+  }
+}
+
+const booksLibrary = new Book();
+booksLibrary.getBookList();
+const formData = document.getElementById('new-book');
+formData.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const getTitle = document.getElementById('title').value;
+  const getAuthor = document.getElementById('author').value;
+  booksLibrary.addNewBook(getTitle, getAuthor);
+});
```

Comment:
- [ ] **[OPTIONAL]** I suggest you clear the input fields after adding a new book so that it tells the user that something actually happened and that book was added successfully. You can do this by resetting the values of the inputs after calling `booksLibrary.addNewBook(getTitle, getAuthor);`. 


```suggestion
formData.addEventListener('submit', (e) => {
  e.preventDefault();
  const getTitle = document.getElementById('title').value;
  const getAuthor = document.getElementById('author').value;
  booksLibrary.addNewBook(getTitle, getAuthor);
  document.getElementById('title').value = ''
  document.getElementById('author').value = ''
});
```